### DISCUSSION TITLE
Filter - Kind

### DISCUSSION THREAD

**Discussion by DerozerDSP** [Geonkick-Synthesizer/geonkick#49] [Show and tell]
Hi dev,

can you write down some additional info about which kind of Filter the Geonkick is using?
I just see two buffers and no coefficients at all on it (except factor for q): basic Moving Average filter?

Thanks

**Comment by DerozerDSP** [comment]
It should be Chamberlin SVF. Thanks!

**Comment by iurienistor** [comment]
@DerozerDSP Hi, sorry, I have never checked the discussion of the repository. Yes, it is digital state variable filter.
